src/enrichment_tables/file.rs

```diff
@@ -111,6 +112,9 @@ pub struct FileConfig {
     /// [rfc3339]: https://tools.ietf.org/html/rfc3339
     /// [chrono_fmt]: https://docs.rs/chrono/latest/chrono/format/strftime/index.html#specifiers
     #[serde(default)]
+    #[configurable(metadata(
+        docs::additional_props_description = "represent mapped log field names and types."
```

Comment:
Nit: ```suggestion
        docs::additional_props_description = "Represents mapped log field names and types."
```

We would also benefit from an example here.